### PR TITLE
Restore Snapcraft files.

### PR DESCRIPTION
**Pull request by jkoritzinsky**
These were accidentally deleted during #38457

### CODE REVIEW

File: src/installer/snaps/dotnet-runtime-3.0/snap/snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: dotnet-runtime-30
```
Comment (jkotas) [inline src/installer/snaps/dotnet-runtime-3.0/snap/snapcraft.yaml:1]: 3.0 is out of support. Do we need to keep it?

Comment (NikolaMilosavljevic) [reply src/installer/snaps/dotnet-runtime-3.0/snap/snapcraft.yaml:1]: > 3.0 is out of support. Do we need to keep it?

@leecow do you know if we need 3.0?

Comment (leecow) [reply src/installer/snaps/dotnet-runtime-3.0/snap/snapcraft.yaml:1]: No need to keep 3.0.

**Review by NikolaMilosavljevic — APPROVED** [review]

File: src/installer/snaps/dotnet-runtime-3.0/snap/snapcraft.yaml (hunk 1 above)
Comment (jkoritzinsky) [reply src/installer/snaps/dotnet-runtime-3.0/snap/snapcraft.yaml]: I've removed the 3.0 snap.

**Pull request merged**